__tests__/plugin.test.ts

```diff
@@ -709,15 +677,15 @@ describe('Auth0Plugin', () => {
   it('should track errors when getAccessTokenWithPopup throws', async () => {
     const plugin = createAuth0({
       domain: '',
-      client_id: ''
+      clientId: ''
     });
 
     plugin.install(appMock);
 
     getTokenWithPopupMock.mockRejectedValue('Some Error');
 
     try {
-      await appMock.config.globalProperties.$auth0.getAccessTokenWithPopup();
+      await appMock.config.globalProperties.$auth0.getAccessTokenWithPopup({});
```

Comment:
Is it expected that we now have to pass `{}` instead of omitting the argument?

Reply:
This was done in error (I think I added this thinking I was in the test above), will remove